applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -75,7 +75,7 @@ static void connected_cb(struct bt_conn *conn, uint8_t err)
 {
 	int ret;
 	char addr[BT_ADDR_LE_STR_LEN];
-	uint8_t num_conn;
+	uint8_t num_conn = 0;
```

Comment:
```suggestion
	uint8_t num_conn = 0U;
```

Reply:
We generally do not use 'U' in the audio application